apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/views/chat/group/GroupChatInfoView.kt

```diff
@@ -219,6 +267,24 @@ fun GroupChatInfoLayout(
       }
       SectionSpacer()
 
+      Row(
+        Modifier
+          .fillMaxWidth()
+          .padding(horizontal = 10.dp),
+        horizontalArrangement = Arrangement.Center,
+        verticalAlignment = Alignment.CenterVertically
+      ) {
+        SearchButton(chat, groupInfo, close)
+        Spacer(Modifier.width(DEFAULT_PADDING))
+        MuteButton(chat, groupInfo)
+        if (groupInfo.canAddMembers) {
+          Spacer(Modifier.width(DEFAULT_PADDING))
+          AddGroupMembersButton(chat, groupInfo)
```

Comment:
perhaps "add members" in the middle and "mute" on the right same as for contact? just seems strange to have "mute" in the middle